Classes/Flight.cs

```diff
@@ -1,27 +1,46 @@
-﻿namespace AirportTicketBooking.Classes
+﻿using ClassType = TicketClassType.ClassType;
+
+namespace AirportTicketBooking.Classes
 {
-    internal class Flight
+    public class Flight
     {
-        internal string FlightNumber { get; set; }
+        internal int FlightNumber { get; set; }
+
         internal string DepartureCountry { get; set; }
+
         internal string DestinationCountry { get; set; }
+
         internal string DepartureAirport { get; set; }
+
         internal string ArrivalAirport { get; set; }
-        internal DateTime LeaveDateTime { get; set; }
+
+        internal DateTime DepartureDate { get; set; }
+
         internal decimal Price { get; set; }
```

Comment:
Why do you have the Price prop here as well? you already have it in the Booking entity.

Reply:
Sorry, I forget to delete this member from here